CHANGELOG.md

```diff
@@ -1,3 +1,154 @@
+# 82.0.0
+
+In this major release, we dropped support for iOS 8 in all components. We also introduced new method and properties for AppBar, BottomSheet and NavigationDrawer.
+
+## Breaking changes
+
+iOS 8 support has been dropped in all components. This change will require application bumping minimum depolyment target to iOS 9.
+
+## New features
+
+### Add support for MDCAppBarNavigationController delegate to pick a tracking scroll view
+
+We added a method in delegate for clients to be able to customize the tracking scroll view detection logic. 
```

Comment:
Tip: Please try to avoid "we", "I", or "you" whenever possible.  You could say,

"An optional delegate method was added to allow clients to specify the controller's tracking scroll view. The delegate may return `nil` if no appropriate tracking scroll view is present."

Reply:
Addressed in 2cb0bf1b20ef7d58d617fc09344b880ffbd376e5.